modules/drivers/druid/src/metabase/driver/druid/query_processor.clj

```diff
@@ -477,7 +478,12 @@
   ([pred output-name] (ag:filtered (parse-filter pred)
                                    (ag:count output-name))))
 
-(defn- create-aggregation-clause [output-name ag-type ag-field]
+(defn- ag:sumWhere
+  ([field pred output-name] (ag:filtered (parse-filter pred)
+                                         (ag:doubleSum field output-name))))
```

Comment:
since there's only one arity to this fn might as well remove the extra `()`

Reply:
Agree. I already have another PR that cleans up the druid code and makes it compliant with our style (#9757 -- but it's 90% just white space things, so probably not worth the review)